plugins/arDominionB5Plugin/scss/_fullwidthtreeview.scss

```diff
@@ -13,27 +13,28 @@
 
 #fullwidth-treeview-row {
   margin-bottom: 30px;
-  border: 1px solid #ddd;
   clear: both;
 }
 
 #fullwidth-treeview {
   overflow-x: hidden;
   height: 100%;
-  background-color: #ffffff;
+  background-color: $white;
+  border: $border-width solid $border-color;
+  border-radius: $border-radius;
```

Comment:
Could we use the "border rounded" classes on the element directly?

Reply:
Also `bg-white` and `mb-3`.

Reply:
Leaving these in the scss file to avoid having to copy hierarchy.js into the B5 theme to make the equivalent changes.